dsn.go

```diff
@@ -36,8 +36,8 @@ var (
 type Config struct {
 	User                 string            // Username
 	Passwd               string            // Password (requires User)
-	Net                  string            // Network type
-	Addr                 string            // Network address (requires Net)
+	Net                  string            // Network protocol
+	Addr                 string            // Network address (requires Net, otherwise defaults to a local connection)
```

Comment:
```suggestion
	Net                  string            // Network (e.g. "tcp", "tcp6", "unix". default: "tcp")
	Addr                 string            // Address (default: "127.0.0.1:3306" for "tcp" and "/tmp/mysql.sock" for "unix")
```

Reply:
https://pkg.go.dev/net#Dial doesn't say neither "network type" nor "network protocol".
It just says "name of the network".

"local connection" is ambiguous.